.github/workflows/ci.yml

```diff
@@ -32,13 +32,15 @@ jobs:
   publish-docker:
     runs-on: ubuntu-latest
     steps:
+      - name: Dump context
```

Comment:
What's the purpose of this step?

Reply:
That dumps the context. Helps in debugging the github context